packages/cadl-python/test/generated/hello/hello/aio/_operations/_operations.py

```diff
@@ -66,12 +68,9 @@ async def world(self, **kwargs: Any) -> str:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
-        if response.content:
-            deserialized = response.json()
-        else:
-            deserialized = None
+        deserialized = _deserialize(None, response.json())
```

Comment:
There is no need to judge `response.content`? Why?

Reply:
Hi @msyyc , @iscai-msft , I added a check for `response.content` before call `_deserialize(...)`